drt_1d.py

```diff
@@ -152,14 +299,16 @@ def perform_calculation(input_data:dict):
     num_voxels = input_data.get("number_of_voxels")
     source_voxel_index = input_data.get("source_voxel_index")
     sigma_s_values = input_data.get("scattering_cross_section")
-
+    positions = input_data.get("positions")
+    
```

Comment:
convert keys to integer right away...
```
    positions = { int(key): value for key, value in positions }
```

(or maybe this is not necessary if you use integer keys to begin with - see below)